src/synapse/workload.py

```diff
@@ -758,6 +758,7 @@ def get_environment(charm_state: CharmState) -> typing.Dict[str, str]:
         # TLS disabled so the listener is HTTP. HTTPS will be handled by Traefik.
         # TODO verify support to HTTPS backend before changing this  # pylint: disable=fixme
         "SYNAPSE_NO_TLS": str(True),
+        "LD_PRELOAD": "/usr/lib/x86_64-linux-gnu/libjemalloc.so.2",
```

Comment:
I'm wondering if we shouldn't validate that the file exists before adding it to preload ? WDYT ?